packages/browser/src/backend.ts

```diff
@@ -79,7 +79,12 @@ export class BrowserBackend implements Backend {
       );
     }
 
-    Raven.config(dsn, this.options).install();
+    Raven.config(dsn, this.options);
+
+    // TODO: Remove once integrations are ported
+    Raven._patchFunctionToString();
```

Comment:
As discussed, these can all be new integrations and call `Raven._xxx();` in `install()`. 
Bonus points for storing internal the old impl and reset it in `uninstall()`.